web/themes/custom/karaoke_2022/components/03-organisms/site/site-header/site-header.twig

```diff
@@ -1,7 +1,7 @@
 {% set header__base_class = 'header' %}
 {# If `directory` is defined, set the path relative for Drupal.
  # Otherwise, set the path relative to the Component Library. #}
-{% set header__logo_src = directory ? '/' ~ directory ~ '/images/logo.svg' : 'logo.png' %}
+{% set header__logo_src = directory  ~ '/images/logo.png' %}
```

Comment:
This breaks the header in Storybook. Did you try just renaming it from `logo.svg` to `logo.png`?

Reply:
Done!